src/main/kotlin/no/skatteetaten/aurora/gobo/integration/boober/AuroraConfigService.kt

```diff
@@ -57,15 +57,15 @@ class AuroraConfigService(
         reference: String,
         fileName: String,
         content: String
-    ): Response<AuroraConfigFileResource> {
+    ): Mono<Response<AuroraConfigFileResource>> {
         val url = "/v2/auroraconfig/{auroraConfig}?reference={reference}"
         val body = mapOf("content" to content, "fileName" to fileName)
 
         return booberWebClient.executeMono<Response<AuroraConfigFileResource>>(token) {
             it.put()
                 .uri(booberWebClient.getBooberUrl(url), mapOf("auroraConfig" to auroraConfig, "reference" to reference))
```

Comment:
Kan ikke booberUrl injectes ett eller annet sted? Litt obskurt å først kalle executeMono, deretter måtte kalle clienten igjen i sin egen lambda for å hente ut url? :thinking: